go/exocom/src/logger/logger.go

```diff
@@ -77,7 +78,7 @@ func (l *Logger) Messages(message structs.Message, originalName string, internal
 			text += fmt.Sprintf("--[ %s ]-[ %s ]->", originalName, internalMessageName)
 		}
 		text += "  " + receiver
-		if message.ResponseTo != "" {
+		if message.ResponseTime > time.Nanosecond {
```

Comment:
Is the default value 0? If so thoughts on comparing  `!= 0`? I think thats a bit more clear